README.md

```diff
@@ -84,43 +84,42 @@ Currently available positions are:
 * `bottom_right_corner`
 * `bottom_left_corner`
 
-7. Configure a local connection to the RabbitMQ by
-- Adding a RabbitMQ URL in your `.env` file:
-```text
-# .env
-
-###> setono/sylius-callouts-plugin ###
-RABBITMQ_URL=amqp://guest:guest@localhost:5672
-###< setono/sylius-callouts-plugin ###
-``` 
-- Adding a [setono_sylius_callouts_plugin.yaml](tests/Application/config/packages/setono_sylius_callouts_plugin.yaml)
-configuration file to your `config/packages` directory.
+7. Configure Async assigns product callouts
 
-```yaml
-imports:
-    - { resource: "@SetonoSyliusCalloutsPlugin/Resources/config/config.yml"}
-
-old_sound_rabbit_mq:
-    connections:
-        default:
-            url: '%env(resolve:RABBITMQ_URL)%'
-    producers:
-        callouts:
-            connection: default
-            exchange_options:
-                name: 'product'
-                type: direct
-    consumers:
-        callouts:
-            connection: default
-            exchange_options:
-                name: 'product'
-                type: direct
-            queue_options:
-                name: 'product'
-            callback: setono_sylius_callouts_plugin.consumer.product_callouts_assigner
-            enable_logger: true
-```
+    This plug assigns product callouts when the administrator create/update the product or callout, with a large number of products this can result in slower page performance. To circumvent that you can use RabbitMQ with Symfony Messenger to assigns product callouts.
```

Comment:
```suggestion
    This plugin assigns product callouts when the administrator create/update the product or callout. With a large number of products this can result in slower page performance. To circumvent this problem you can use an async transport with Symfony Messenger to assigns product callouts.
```